Makefile

```diff
@@ -88,3 +91,15 @@ cover:
 		tail -n +2 cover.out >> cover-all.out;)
 	go tool cover -html=cover-all.out
 .PHONY: cover
+
+dockerbuild:
+	docker build -t $(DOCKER_IMAGE_NAME):latest .
+.PHONY: dockerbuild
+
+dockerpush: dockerbuild
+	echo "$(DOCKER_PASSWORD)" | docker login -u "$(DOCKER_USERNAME)" --password-stdin
+	docker tag $(DOCKER_IMAGE_NAME) spacemeshos/$(DOCKER_IMAGE_NAME):develop
```

Comment:
so we're tagging both with SHA1 and the branch name right ? better to replace `develop` with `$(BRANCH)` since we have that already.